UI/ChatUI/ChatUI/ChatUI/MainPage.xaml

```diff
@@ -0,0 +1,427 @@
+﻿<Page
+	x:Class="ChatUI.MainPage"
+	xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+	xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+	xmlns:android="http://uno.ui/android"
+	xmlns:behaviors="using:ChatUI.Behaviors"
+	xmlns:converters="using:ChatUI.Converters"
+	xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+	xmlns:ios="http://uno.ui/ios"
+	xmlns:local="using:ChatUI"
+	xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+	xmlns:win="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+	xmlns:xamarin="http://uno.ui/xamarin"
+	Background="{ThemeResource ApplicationPageBackgroundThemeBrush}"
+	mc:Ignorable="d ios android xamarin">
+
+	<Page.Resources>
+		<ResourceDictionary>
+			<converters:FromNullableBoolToVisibilityConverter x:Key="BooleanToVisibility" VisibilityIfTrue="Visible" />
+
+			<Style TargetType="ListView">
```

Comment:
I think that we should extract this template in the toolkit in order to make it easy to re-use in apps.

Reply:
Agreed